package.json

```diff
@@ -26,8 +26,8 @@
     "test:ci": "jest --coverage --runInBand"
   },
   "engines": {
-    "node": ">= 6.x <= 9.x",
-    "npm": ">= 2.x <= 5.x",
+    "node": ">= 6.x <= 10.x",
```

Comment:
Should we remove the upper bound directly? Is it possible to do so?

Reply:
Yes, it's possible and IMO preferable unless there's an specific reason to have it.

Reply:
Done! Also got rid of the `.x` as a partial version range is already treated as an X-Range.